lib/dialects/mssql/query.js

```diff
@@ -211,6 +213,11 @@ class Query extends AbstractQuery {
     });
   }
 
+  handleShowConstraintsQuery(data) {
```

Comment:
@sushantdhiman Keeping this as i've modified `handleShowConstraintsQuery` to include converting snake_case keys to camelCase.